sympy/printing/tests/test_latex.py

```diff
@@ -504,6 +504,13 @@ def test_latex_union():
         r"\left\{1, 2\right\} \cup \left[3, 4\right]"
 
 
+def test_latex_symmetric_difference():
+    assert latex(SymmetricDifference(Interval(2, 5), Interval(3, 5))) == \
+        r"\left[2, 3\right)"
+    assert latex(SymmetricDifference(Interval(1,10), Interval(2,20))) == \
```

Comment:
This `SymmetricDifference` evaluates to union of two intervals, you have to test if unevaluated `SymmetricDifference`s are being printed correctly.